pp_ctl.c

```diff
@@ -5434,6 +5457,47 @@ PP(pp_break)
     return cx->blk_givwhen.leave_op;
 }
 
+static void
+invoke_defer_block(pTHX_ void *_arg)
+{
+    OP *start = (OP *)_arg;
+    I32 was_cxstack_ix = cxstack_ix;
```

Comment:
this variable is unused in non-DEBUGGING builds

Reply:
Ahyes. Now fixed in a488435fb4. (I'll squash before merging)